packages/framework/src/Component/Error/ErrorIdProvider.php

```diff
@@ -1,31 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Shopsys\FrameworkBundle\Component\Error;
-
-use Shopsys\FrameworkBundle\Component\String\HashGenerator;
-
-class ErrorIdProvider
-{
-    protected ?string $errorId = null;
-
-    /**
-     * @param \Shopsys\FrameworkBundle\Component\String\HashGenerator $hashGenerator
-     */
-    public function __construct(protected readonly HashGenerator $hashGenerator)
-    {
-    }
-
-    /**
-     * @return string
-     */
-    public function getErrorId(): string
```

Comment:
I don't think we should remove error id from the administration

Reply:
Makes sense. I rewrote that feature. Now `errorId` is automatically provided in every exception that is thrown and also sent to Sentry as a custom tag. That means, you can filter by `errorId` in Sentry :)

<img width="821" alt="Snímek obrazovky 2025-02-27 v 18 41 25" src="https://github.com/user-attachments/assets/a89a6b36-3781-492e-8375-ed80b5f8b9be" />